airstack-modules/modules/airstack/domain.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Address,
+  BigInt,
+  dataSource,
+  log,
+} from "@graphprotocol/graph-ts";
+
+import {
+  AirAccount,
+  AirEntityCounter,
+  AirToken,
+  AirBlock,
+  AirMeta
+} from "../../generated/schema";
+
+import { AIR_META_ID, BIGINT_ONE, SUBGRAPH_SCHEMA_VERSION, SUBGRAPH_VERSION, SUBGRAPH_NAME, SUBGRAPH_SLUG, processNetwork } from "./utils";
+
+export namespace domain {
+  export function trackAirDomainTransferTransaction(): void {
+
+  }
+
+  export function updateAirEntityCounter(
```

Comment:
good idea, but this would also expose the function for people to use outside the module and they could potentially use it to manipulate entity counters.

Reply:
but, they can do it without these functions also, as they will have full access to these entities eitherways, so, I am making this change.

Reply:
done